src/routes/webhooks/paddle.ts

```diff
@@ -74,6 +82,39 @@ const updateUserSubscription = async ({
     );
     return false;
   }
+
+  const { gifter_id: gifterId, duration } = customData;
```

Comment:
Shouldn't we extract duration from the actual message?
So it will say start date or something like that of purchase and API we can just do + 1 year?

Reply:
I was worried if we pass to frontend user can inject it to whatever so might need to be set fixed on the API https://github.com/dailydotdev/daily-api/pull/2603#discussion_r1922274898